### ISSUE TITLE
HPF | issues with 'set' method

### ISSUE THREAD

**Issue by mariobuoninfante** [ccrma/chuck#132]
Hi,

With HPF, 'set' method seems to behave differently from 'freq'.
The following code shows the issue
```
Noise noise => HPF filter => dac;

100 => float f;
filter.Q(1);

while( true )
{
    f + 100 => f;
    if( f > 5000 )
    {
        100 => f;
    }
    // swap between filter.freq and filter.set to check the difference
    filter.freq(f);
    //filter.set( f, 1 );
    0.1::second => now;
}
```

tested on ChucK 1.4.0.0 - Ubuntu Studio 16.04 64bit

**Comment by lathertonj** [comment]
Yup, looks like there is a bug where the HPF set() function calls set_rlpf internally instead of set_rhpf.

**Comment by lathertonj** [comment]
Fixed in 29aede50c6dd7a81fdb7074f7cb400bc23c50c4a.
